_posts/2020-02-14-numfocus-sponsored-project.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: MDAnalysis is a NumFOCUS Sponsored Project!
+---
+
+<img src="{{ site.baseurl }}{{ site.images }}/numfocus-sponsored.png"
+style="float: right" alt="NumFOCUS Sponsored" width="40%"/>
+
+**[MDAnalysis is now a NumFOCUS Sponsored Project]({{
+site.numfocus.sponsored_project }})!**
+
+Let us explain who MDAnalysis and NumFOCUS are and what being a
+sponsored project means:
+
+[**MDAnalysis**]({{ site.baseurl }}/about) is an open source software
+project that produces the *MDAnalysis Python library* for the analysis
+of computer simulations of many-body systems at the molecular scale,
+spanning use cases from interactions of drugs with proteins to novel
+materials. It is widely used in the scientific community and is
+written by scientists for scientists. The MDAnalysis Project is
+represented by the [MDAnalysis Core Developers]({{ site.baseurl
+}}/about#governance).
```

Comment:
As Max suggests, maybe a simple text list for "core devs" would be appropriate if you want something to link to.

Reply:
added to About